e2e/wdio/core/pages/alert.po.ts

```diff
@@ -7,6 +7,9 @@ export class AlertPo extends CoreBaseComponentPo {
 
     button = ' button';
     alerts = 'fd-alert-example .fd-alert';
+    playgroundAlert = '.fd-playground .fd-alert ';
```

Comment:
Do you need space at the end of the selector?

Reply:
yeah, in other case test can fail due to incorrect selector

Reply:
Ok